src/nyc_trees/apps/event/templates/event/partials/event_section.html

```diff
@@ -1,4 +1,6 @@
 {% with event=info.event %}
+{% block extra_markup %}
+{% endblock extra_markup %}
```

Comment:
It's actually so that templates that inherit from this one, can inject fields here. This happens on the "edit user setting page", which displays the event list with the addition of a checkbox next to each event (for the email opt-in).